packages/sonarwhal/src/lib/sonarwhal.ts

```diff
@@ -85,6 +89,33 @@ export class Sonarwhal extends EventEmitter {
         });
     }
 
+    private resolveRule(key: string): string {
```

Comment:
Not a fan of this part. It's weird to have to do all of this and it will slow down the start up process. We need to find another way.
Maybe `resources` should use the value in `.sonarwhalrc` in the values it returns. We can access the `id` via `meta.id` and do the matching from there.